src/SentryTarget.php

```diff
@@ -163,15 +181,75 @@ public function export()
                     $event = Event::createEvent();
                     $event->setMessage($data['message']);
                     $event->setLevel($this->getLogLevel($level));
+                    $event->setTimestamp($timestamp);
 
                     \Sentry\captureEvent($event, EventHint::fromArray(array_filter([
                         'exception' => $data['exception'] ?? null,
+                        'extra' => $data['extra'] ?? [],
```

Comment:
Please check this line, it seems not the same extra data that we can attach to scope

Reply:
All scope `extra` data will be directly copied into `extra` field of the each event in [this method](https://github.com/getsentry/sentry-php/blob/95fd4c2d723b7e84a2ced610989ae59a467f1a73/src/State/Scope.php#L339), so both extra arrays has same format